src/components/fragment.js

```diff
@@ -50,7 +50,15 @@ const relativePaths = (ComposedComponent: ReactClass<*>) => {
 
       const location = store.getState().router;
 
-      const routePrefix = parentRoute && parentRoute !== '/' ? parentRoute : '';
+      const routePrefix = parentRoute && parentRoute !== '/'
+        ? parentRoute
```

Comment:
Nit: this is actually my preferred style for ternaries in general, but since both branches are so short, I think we can do

```js
const routePrefix = parentRoute && parentRoute !== '/'
  ? parentRoute : ''
```